ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -440,6 +472,108 @@ export default {
+            return;
+
+          // Center = 0, -75
+          const x = statueData.combatants[0].PosX;
+          const y = statueData.combatants[0].PosY - -75;
+          statue.dist = Math.hypot(x, y);
+          if (data.statues.length > 1) {
+            // Sort so that closest statue (inner) is first
+            data.statues.sort((a, b) => a.dist - b.dist);
+          }
+        }
+      },
+      infoText: (data, matches, output) => {
+        // Player is the source of the tether for this one, oddly enough
+        if (data.me !== matches.source)
+          return;
+
+        if (data.statues.length > 0 && data.statues[0].id === matches.targetId)
+          data.statueDir = 'inner';
+        else if (data.statues.length > 1 && data.statues[1].id === matches.targetId)
+          data.statueDir = 'outer';
+        else
+          data.statueDir = 'unknown';
+
+        return output[data.statueDir]();
+      },
+      outputStrings: {
+        inner: {
+          en: 'Inner Statue',
+        },
+        outer: {
+          en: 'Outer Statue',
+        },
+        unknown: {
+          en: '???',
+        },
+      },
+    },
+    {
+      id: 'E12S Promise Palm Of Temperance SE',
+      netRegex: NetRegexes.startsUsing({ source: 'Guardian Of Eden', id: '58B4', capture: false }),
+      durationSeconds: 10,
+      infoText: (_1, _2, output) => output.knockback(),
+      outputStrings: {
+        knockback: {
+          en: 'Knockback SE',
+        },
+      },
+    },
+    {
+      id: 'E12S Promise Palm Of Temperance SW',
+      netRegex: NetRegexes.startsUsing({ source: 'Guardian Of Eden', id: '58B5', capture: false }),
+      durationSeconds: 10,
+      infoText: (_1, _2, output) => output.knockback(),
+      outputStrings: {
+        knockback: {
+          en: 'Knockback SW',
+        },
+      },
+    },
+    {
+      // Time the first dodge for lasers from the Rapturous Reach left/right split
+      id: 'E12S Promise Statue Laser First',
+      netRegex: NetRegexes.startsUsing({ source: 'Eden\'s Promise', id: '58A[DE]', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Edens Verheißung', id: '58A[DE]', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Promesse D\'Éden', id: '58A[DE]', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'プロミス・オブ・エデン', id: '58A[DE]', capture: false }),
+      // Check against `data.statueDir` to limit this trigger to just the Chiseled Sculpture phase
+      condition: (data) => data.statueDir !== undefined,
+      durationSeconds: 5,
+      response: statueBaitResponse,
+    },
+    {
+      // Time the other dodges from the previous lasers going off
+      id: 'E12S Promise Statue 2nd/3rd/4th Laser',
+      netRegex: NetRegexes.ability({ source: 'Chiseled Sculpture', id: '58B3', capture: false }),
+      condition: (data) => (console.log(data) || true) && data.statueLaserCount < 4,
```

Comment:
Can you remove this debugging line?

Reply:
Removed for next commit. Apparently when I ran a search for it before, I typo'd console 🤷